man/subscription-manager.8

```diff
@@ -92,49 +92,46 @@ must be passed as system arguments in a non-interactive session.
 7. import
 
 .IP
-8. redeem
```

Comment:
Please, just remove redeem command from the list. There will be lot of removals of commands. Lot of conflicts. We will fix numbers in final edit of man page.